Scenarios/TESTScenarios.json

```diff
@@ -27,7 +27,7 @@
   ],
   "scenarios": [
     {
-      "name": "GQ -> C(45%) -> GQ -> C(50%)",
+      "name": "PageView",
```

Comment:
Why did all of this file change? Anyone previously using this scenario will now log PageView instead of clicks.

Wouldn't it be better to create a new one?